File: 27. LeetCode Problems/Two-Sum.py
def twoSum(self, nums: List[int], target: int) -> List[int]:
  #  created a dictionary to store the occurrences of t         
  dic = {}
  n = len(nums)
  for i in range(n):
      # checking for the occurrence of the target number         
      if target - nums[i] in dic:
          return [ dic[ target - nums[i]],i ]
      dic[nums[i]] = i        

File: 27. LeetCode Problems/two_sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int>v;
        int n=nums.size();
        for(int i=0;i<n-1;i++)
        {
            for(int j=i+1;j<n;j++)
            {
                if(nums[i]+nums[j]==target)
                {
                    v.push_back(i);
                    v.push_back(j);
                 }
            }
        }
        
        return v;
    }
};